app/serializers/technology_serializer.rb

```diff
@@ -1,3 +1,14 @@
 class TechnologySerializer < ActiveModel::Serializer
   attributes :id, :name
+
+  def name
+    tech_capitalizations[object.name] ?
+    tech_capitalizations[object.name] : object.name.capitalize
+  end
+
+  def tech_capitalizations
+    {
+      "javascript" => "JavaScript",
```

Comment:
I guess we'll only add stacks here that have weird capitalizations? Otherwise just the first letter will be capitalized. Are we looking to add more tech types here eventually? @hhoopes